exec/java-exec/src/test/resources/regex/simple.log2

```diff
@@ -0,0 +1,5 @@
+2017-12-17 10:52:41,820 [main] INFO  o.a.d.e.e.f.FunctionImplementationRegistry - Function registry loaded.  459 functions loaded in 1396 ms.
```

Comment:
As far as I can tell, these two files are identical. I think you got these from my example, so that is probably my fault; I guess I didn't realize that they were the same...

We really only need one of them.

Reply:
Removed log2.